src/utils/urlPaths.ts

```diff
@@ -85,10 +85,17 @@ export const getRegistrationLandingPagePath = (identifier: string) =>
 export const getImportCandidatePath = (identifier: string) =>
   UrlPathTemplate.BasicDataCentralImportCandidate.replace(':identifier', encodeURIComponent(identifier));
 
-export const getRegistrationWizardPath = (identifier: string, tab?: number): To => {
+export const getRegistrationWizardPath = (identifier: string, tab?: number, shouldNotRedirect = false): To => {
```

Comment:
kanskje endre de to siste her til objekt også, så man slipper å sende inn `undefined` som argument for å få til det man vil